config/core/test-configs-chromeos.yaml

```diff
@@ -77,7 +77,7 @@ device_types:
       - passlist: {defconfig: ['x86-chromebook']}
     params: &octopus-params
       cros_flash_nfs:
-        base_url: 'https://storage.staging.kernelci.org/images/rootfs/debian/bullseye-cros-flash/20220523.1/amd64/'
+        base_url: 'https://storage.kernelci.org/images/rootfs/debian/bullseye-cros-flash/20220527.0/amd64/'
```

Comment:
@gctucker Is there any reason why the URL is using HTTPS?

Reply:
@mgalka So do you think it should be HTTP?

Reply:
I don't mind. I was just curious.